README.md

```diff
@@ -40,10 +40,9 @@ Authorization](https://cloud.google.com/binary-authorization/).  For details see
 
 * Watch the talk on [Software Supply Chain Management with Grafeas and Kritis](https://www.infoq.com/presentations/supply-grafeas-kritis/)
 * Learn the concepts in the [Kritis whitepaper](docs/binary-authorization.md)
```

Comment:
Minor: these are all full sentences, maybe add "." at the end of each.

Reply:
great point, done